examples/AGS02MA_calibrate_manual/AGS02MA_calibrate_manual.ino

```diff
@@ -0,0 +1,136 @@
+//
+//    FILE: AGS02MA_calibrate_manual.ino
+//  AUTHOR: Rob Tillaart, Beanow
+// VERSION: 0.2.0
+// PURPOSE: test application
+//    DATE: 2022-04-22
+//     URL: https://github.com/RobTillaart/AGS02MA
+//
+
+#include "AGS02MA.h"
+
+// The zero calibration value we'll (temporarily) set in the example.
+#define ZC_VALUE 700
+
+#define READS 10
+#define INTERVAL 3000
+
+
+AGS02MA AGS(26);
+
+AGS02MA::ZeroCalibration initialValue;
+u_int8_t version;
```

Comment:
should be uint8_t

Reply:
Ha, oops. Fixed that.

Reply:
Weird how `u_int8_t` even compiled for me. I'm guessing the arduino-esp32 fork includes it somehow while not being in the base Arduino. Because the CI test does fail to compile on that. :shrug:

Reply:
`F12` to the rescue, indeed the culprit was this :eyes:
`~/.arduino15/packages/esp32/tools/xtensa-esp32-elf-gcc/gcc8_4_0-esp-2021r2/xtensa-esp32-elf/sys-include/sys/types.h`